arb_os/arbsys.mini

```diff
@@ -115,7 +116,25 @@ impure func arbsys_withdrawErc20(topFrame: EvmCallFrame, calldata: ByteArray) {
 
     let messageData = bytearray_extract(calldata, 4, 64);
     queueMessageForSend(1, senderAddr, messageData);
-    evmOp_return(0, 0);
+    if (evmCallStack_addEvmLogToCurrent(
+        (
+            address(100),  // addEvmLogToCurrent will fill this in
+            bytearray_set256(
```

Comment:
This is currently expected to be marshalled bytes so this and the others need a `bytearray_marshalFull`

Reply:
Good catch.  Fixed.